.eslintrc.cjs

```diff
@@ -11,6 +11,27 @@ module.exports = {
 	rules: {
 		// add rules... or dont...
 		'no-debugger': 'error',
-		'@typescript-eslint/ban-ts-comment': 'off'
+		'@typescript-eslint/ban-ts-comment': 'off',
+		'@typescript-eslint/no-explicit-any': 'off',
+		'@typescript-eslint/no-non-null-assertion': 'off',
+		'@typescript-eslint/no-non-null-asserted-optional-chain': 'off',
+		'@typescript-eslint/no-empty-function': 'off',
```

Comment:
`no-empty-function` - we should consider enabling this as well, I suspect only our tests would be using empty functions, but if used in core code I'd question why they exist.